recipes/bshoshany-thread-pool/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class TestPackageConan(ConanFile):
+    settings = "arch", "build_type", "compiler", "os"
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package_multi"
```
Missing generator which should create bshoshany-thread-pool-config.cmake